moodler/moodle.py

```diff
@@ -164,14 +173,18 @@ def export_grades(course_id, output_path, should_export_feedback=False):
     course_name = locate_course_name(course_id)
     session = connect_to_server(MOODLE_USERNAME, MOODLE_PASSWORD)
 
-    grades_spreadsheet_file = download_course_grades_report(
-        course_id,
-        course_name,
-        should_export_feedback,
-        output_path,
-        session)
+    try:
+        grades_spreadsheet_file = download_course_grades_report(
+            course_id,
+            course_name,
+            should_export_feedback,
+            output_path,
+            session)
+
+        logger.info("Downloaded the file '%s' as the grades exported spreadsheet", grades_spreadsheet_file)
+    except DownloadException:
```

Comment:
You are hiding which type of DownloadException has been raised. You should use the following for instance:
except DownloadException as e:
Then, you are able to print the very exception that was raised. If you do not want this error to crash the program, log the exception message itself to the log. But do not hide information.